generate_shuffle_tables.py

```
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+"""
+Generate shuffle tables for StreamVByte x64 SIMD decoding.
+Generates both 0124 and 1234 variants.
+"""
+
+def generate_length_table_0124():
```

Ruff (0.12.2)

Missing return type annotation for public function `generate_length_table_0124`

(ANN201)